docs/topics/GATEWAY.md

```diff
@@ -196,16 +193,6 @@ Sent when a client wants to join, move, or disconnect from a voice channel.
 }
 ```
 
-### Gateway Reconnect
-
-Used to redirect clients to a new gateway. Clients should disconnect from the existing gateway, connect to the provided new gateway, and [resume their session](#DOCS_GATEWAY/gateway-resume) on the new gateway.
-
-###### Gateway Reconnect Structure
-
-| Field | Type | Description |
-|-------|------|-------------|
-| url | string | url to new gateway |
```

Comment:
Sorry if I was unclear. Gateway reconnect is still a thing. We just don't send the `url` anymore. The client should try to connect to the last discovered gateway URL (and if it fails, re-discover it using /api/gateway)